src/prop-types/childrenOf.js

```diff
@@ -1,4 +1,5 @@
 import { Children } from 'react';
+import { areComponentsEqual } from 'react-hot-loader/patch';
```

Comment:
@joshblack Would this add `react-hot-loader` dependency to our main React component library?

Reply:
Looking at this, it seems so. Leaving a comment in the general thread.